Dockerfile

```diff
@@ -73,7 +73,7 @@ RUN LIBRARY_PATH=/lib:/usr/lib /bin/sh -c "/venv/bin/pip install setuptools==33.
 # Install `libmagic` (needed since rapidpro v3.0.64)
 RUN apk add --no-cache postgresql-client libmagic
 
-ENV UWSGI_VIRTUALENV=/venv UWSGI_WSGI_FILE=temba/wsgi.py UWSGI_HTTP=:8000 UWSGI_MASTER=1 UWSGI_WORKERS=8 UWSGI_HARAKIRI=20
+ENV UWSGI_VIRTUALENV=/venv UWSGI_WSGI_FILE=temba/wsgi.py UWSGI_HTTP=:8000 UWSGI_MASTER=1 UWSGI_WORKERS=8 UWSGI_HARAKIRI=60
```

Comment:
If this is just an env var, can't it be overwritten in the docker env config? Or is the goal here to overwrite the default for all images?

Reply:
Let me try it, would be nice if I don't have to build this.

Reply:
The setting works, still getting timeouts but will need to take a different route